### ISSUE TITLE
Running hariko server from a file returns [object Object]

### ISSUE THREAD

**Issue by folmert** [rymizuki/node-hariko#11]
After a lot of pain caused by failing installation of node-gyp I finally installed hariko, but when I run:

```
hariko -f API.md -p 8080 -w
```

it returns:
```
var data = protagonist.parseSync(rawdata, {type: 'ast'});
                         ^
[object Object]
```

I'm running on:
- Windows 10 64bit
- node v.8.7.0
- npm v.5.4.2

**Comment by rymizuki** [comment]
oh... I will take a look.

**Comment by rymizuki** [comment]
Because I did not think about using it on windows, it may take a while.

**Comment by rymizuki** [comment]
I confirmed the same problem with the procedure.

```
npm install --global windows-build-tools
npm install hariko
npm run hariko -- -f ./API.md -p 8080 -w
```

**Comment by rymizuki** [comment]
@folmert fixed issue and published [v1.1.7](https://github.com/rymizuki/node-hariko/releases/tag/v1.1.7).
please retry install and run.

**Comment by folmert** [comment]
It works now, thanks!
